packages/cozy-doctypes/src/Document.js

```diff
@@ -201,20 +201,28 @@ class Document {
 
   /**
    * Update a document with `update` attributes. If the
-   * `update` does not concern "important" attributes, the original
+   * `update` does not concern "important" attributes
+   * or is not forced with forceUpdate option, the original
    * document is returned. Otherwise, the update document is
    * returned with metadata updated.
    *
+   * @param {object} doc - The document already existing in db
+   * @param {object} update - The update to apply to the document
+   * @param {object} options - Options object
+   * @param {boolean} options.forceUpdate - Should the method force the update even if important attributes are identical
+   * @returns {object} - The updated document with cozy new metadata when an update has been done
+   *
```

Comment:
Thanks for the jsdoc!
I would just have changed 2 things:
- Describe a bit more what an "important" attributes is, and what it means. From an external reader, "important" is bit vague. Is "deduplication attributes" relevant, for instance?
- I feel like this jsdoc should especially be on the `bulkSave` method, as this is the actual API called by the app/konnector